librz/core/cfile.c

```diff
@@ -1338,7 +1337,7 @@ RZ_API int rz_core_file_list(RzCore *core, int mode) {
 				free(absfile);
 			}
 			break;
-		case 'n': {
+		case RZ_OUTPUT_MODE_LONG: {
```

Comment:
this seems to be RZ_OUTPUT_MODE_RIZIN, not LONG

Reply:
It's already been used before and the output is different.

Reply:
create a separate function for this. It's probably not even tested anyway, but it does not seem related to the other modes. THis is indeed a very separate thing, which deserves its own function.

Reply:
Done